src/tests/ftest/soak/soak.py

```diff
@@ -828,9 +835,9 @@ def run_soak(self, test_param):
             self.log.info(
                 "<<PASS %s completed in %s >>", self.loop, DDHHMMSS_format(
                     loop_time))
-            # if the time left if less than a loop exit now
-            if end_time - time.time() < loop_time:
-                break
+            # # if the time left if less than a loop exit now
```

Comment:
```suggestion
            # # if the time left is less than a loop exit now
```

Reply:
I will make the change in a future PR; thanks for catch that...